### ISSUE TITLE
RabbitMQ 3.7.0 OS X standalone package segfaults on OS X 10.13 High Sierra

### ISSUE THREAD

**Issue by jamieps** [rabbitmq/rabbitmq-server#1450]
RabbitMQ 3.7.0 Mac standalone package with Erlang 19.3
OS X 10.13.1 - High Sierra

TL;DR

To reproduce:
* Install RabbitMQ from `rabbitmq-server-mac-standalone-3.7.0.tar.xz`
* Run `rabbitmq-server`
* Run any `rabbitmqctl` command, e.g. try and add a user
* Observe that `rabbitmq-server` segfaults with:
```
Assertion failed: (ctx), function digest_update, file /BuildRoot/Library/Caches/com.apple.xbs/Sources/boringssl/boringssl-109.20.5/apple/crypto/digests.c, line 49.
```

---
In High Sierra Apple removed OpenSSL and replaced it with LibreSSL.

Using the OS X standalone package for 3.7.0, `rabbitmq-server-mac-standalone-3.7.0.tar.xz`, results in `rabbitmq-server` aborts on startup because it is dynamically linking against LibreSSL rather than OpenSSL from Homebrew:

```
completed with 3 plugins.
Assertion failed: (ctx), function digest_update, file /BuildRoot/Library/Caches/com.apple.xbs/Sources/boringssl/boringssl-109.20.5/apple/crypto/digests.c, line 49.
./rabbitmq-server: line 318:  6093 Abort trap: 6           start_rabbitmq_server "$@"
```

There is an Erlang bug raised for this - https://bugs.erlang.org/browse/ERL-439 - which disables dynamic linking so that it uses OpenSSL from Homebrew. According to that bug it was backported to Erlang 19.3, however this doesn't seem to have made it into the Erlang included in the RabbitMQ 3.7.0 package.

Please could you update the Erlang included in the OS X standalone package for 3.7.0 _or_ place a warning on https://www.rabbitmq.com/install-standalone-mac.html that it will not work with High Sierra?

We have worked around this by using RabbitMQ from Homebrew, which has corrected this - https://github.com/Homebrew/homebrew-core/pull/15297.

There is also some further background here: https://github.com/elixir-lang/elixir/issues/6200

**Comment by michaelklishin** [comment]
This belongs to https://github.com/rabbitmq/rabbitmq-server-release.
